code/modules/uplink/uplink_items/uplink_devices.dm

```diff
@@ -240,7 +240,8 @@
 	desc = "A disk containing the procedure to perform a brainwashing surgery, allowing you to implant an objective onto a target. \
 	Insert into an Operating Console to enable the procedure."
 	item = /obj/item/disk/surgery/brainwashing
-	cost = 3
+	restricted_roles = list("Medical Doctor", "Roboticist")
```

Comment:
Can you place this stuff in the correct file and change the name path so it shows in "restricted roles" tab?

Reply:
Thanks for the reminder